packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js

```diff
@@ -74,12 +78,28 @@ class Banner extends Component {
         show={show}
       >
 
+        {onPrevious &&
+          <PlainButton
+            color={color}
+            icon={<CaretPrevious color={color} size='small' />}
```

Comment:
@seanmiller26 this is an example of something that is undocumented and where we could improve our theme as I mentioned when I first reviewed the design system with you.

Reply:
I avoided pixel values because they’re historically inaccessible for people with poor vision, who might wish to render the icons larger than 15px. 

That’s probably not true in modern browsers but ems would definitely respect browser settings, where people have set the browser to use larger text sizes.

Reply:
Feel free to investigate the issue with grommet-icons and/or update the theme for button icons, but this should match the equivalent of 1em or 15px for our styles. Right now, `small` I think corresponds with the grommet default of 12px.

Reply:
So I bumped up the minimum text size in Firefox and I think we might fail [WCAG 1.4.4](https://www.w3.org/WAI/GL/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html). Text increases in size, as do the image toolbar icons, but fixed height icons like the subject metadata stay tiny. The legibility of those icons at large text sizes is outside the scope of this PR, but I don't want to add inaccessible code here.
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/59547/133224300-035ef812-5bf8-437a-9250-ad0f7550fa31.png">

Reply:
Page Zoom seems fine up to 200%, though. The page only breaks if I set a minimum font size that's larger than the default of 16. So maybe this would pass? 🤷‍♂️